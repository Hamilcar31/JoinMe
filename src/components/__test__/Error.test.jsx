import React from 'react';
import renderer from 'react-test-renderer';

import ErrorMessage from '../Error';

describe('Error component', () => {
  it('with default parameters', () => {
    // given
    const errorMessage = <ErrorMessage />;

    // when
    const tree = renderer.create(errorMessage).toJSON();

    // then
    expect(tree).toMatchSnapshot();
  });
});
