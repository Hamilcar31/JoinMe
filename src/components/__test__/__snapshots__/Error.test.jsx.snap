// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component with default parameters 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "margin": 10,
    }
  }
/>
`;
